ui/src/album/AlbumGridView.js

```diff
@@ -156,43 +194,78 @@ const AlbumGridTile = ({ showArtist, record, basePath }) => {
           sortBy={'maxYear'}
           sortByOrder={'DESC'}
           className={classes.albumSubtitle}
+          dataKey={'maxYear'}
```

Comment:
What is the reasoning to use `dataKey` here?

Reply:
`dataKey` is used in placed where `source` doesn't match the entry in the record. In this case, there's no value called `year`, but `maxYear` in the record. It's not used for displaying, but for checking if its been loaded since react-virtualized gets the `cellData` from `record[dataKey]`.

https://github.com/samarsault/navidrome/blob/f684c9f0357965afa24fafaca7515c07049a58b9/ui/src/infiniteScroll/VirtualTable.js#L60-L69